gapic/src/main/com/google/gapid/models/Thumbnails.java

```diff
@@ -58,72 +47,27 @@
       .setMaxHeight(DPIUtil.autoScaleUp(THUMB_SIZE))
       .setWireframeMode(Service.WireframeMode.None)
       .build();
+  private static final Service.UsageHints HINTS = Service.UsageHints.newBuilder()
+      .setPreview(true)
+      .build();
 
   private final Client client;
   private final Devices devices;
   private final AtomStream atoms;
-  private final ListenerCollection<Listener> listeners = Events.listeners(Listener.class);
-  private final Queue<Thumbnail> queuedThumbnails = new PriorityQueue<>();
-  private final Thread processorThread;
 
-  public Thumbnails(Client client, Devices devices, Capture capture, AtomStream atoms) {
+  public Thumbnails(Client client, Devices devices, AtomStream atoms) {
     this.client = client;
     this.devices = devices;
     this.atoms = atoms;
-    this.processorThread = new ProcessorThread(client, queuedThumbnails);
-
-    devices.addListener(new Devices.Listener() {
```

Comment:
This is still required. Are we not still planning on allowing the user to change the replay device?